docs/dev-environments/ethereum/ganache/cleanup.sh

```diff
@@ -13,44 +13,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-check_and_exit()
+function error_exit()
 {
-    if [ $? -ne 0 ]
-    then
-        echo "Script failed."
-        exit 1
-    else
-        echo "Done."
-    fi
+    echo "[Error]: " ${*}
+    exit 1
 }
 
 echo ""
 echo "=================================================="
 echo "STEP 1 :: Stop Ganache cli and destroy container"
 echo "=================================================="
-docker stop local-ganache
-docker rm local-ganache
-check_and_exit
+docker stop local-ganache || error_exit "Failed to stop Ganache container"
+echo "Done"
+docker rm local-ganache || error_exit "Failed to delete Ganache container"
+echo "Done"
 
 echo ""
 echo "=================================================="
 echo "STEP 2 :: Stop truffle setup container and destroy"
 echo "=================================================="
-docker-compose -f ./docker-compose-truffle.yaml down
-check_and_exit
+docker-compose -f ./docker-compose-truffle.yaml down || error_exit "Failed to cleanup truffle setup"
```

Comment:
either change ./docker-compose-truffle.yaml -> docker-compose-truffle.yaml
else give full path from $TCF_HOME

Reply:
done. updated to docker-compose-truffle.yaml removing the preceeding ./